packages/experiments/src/components/SelectedItemsList/SelectedPeopleList/SelectedPeopleList.tsx

```diff
@@ -93,10 +93,28 @@ export class SelectedPeopleList extends BasePeopleSelectedItemsList {
       );
     } else {
       let onRenderItem = this.props.onRenderItem as (props: ISelectedPeopleItemProps) => JSX.Element;
-      return onRenderItem({ ...props });
+      let renderedItem = onRenderItem(props);
+      return (
+        props.menuItems.length > 0 ?
+          (
+            <SelectedItemWithContextMenu
+              renderedItem={ renderedItem }
+              beginEditing={ this._beginEditing }
+              menuItems={ this._createMenuItems(props.item) }
+              item={ props.item }
+            />
+          )
+          : renderedItem
+      );
     }
   }
 
+  @autobind
+  private _beginEditing(item: IExtendedPersonaProps): void {
```

Comment:
Why use begin and complete editing? You should be able to pass a dismiss function to contextualmenu that will handle some of this.

Reply:
The contextual menu will not be open throughout the duration of this, so I don't think using this dismiss function is correct here.